Utils/src/PhotonIDisoProducer.cc

```diff
@@ -239,39 +269,44 @@ PhotonIDisoProducer::produce(edm::StreamID, edm::Event& iEvent, const edm::Event
     if(isBarrelPhoton || isEndcapPhoton){
       passAcc=true;
     }
-    
-    // apply id cuts using https://twiki.cern.ch/twiki/bin/view/CMS/CutBasedPhotonIdentificationRun2#Recommended_Working_points_for_2
-    if (isBarrelPhoton) {
-      if (iPhoton->hadTowOverEm() < 0.0597 && !hasMatchedPromptElectron(iPhoton->superCluster(), electrons, conversions, beamSpot->position())) {
-	passIDLoose = true;
-	if (sieie < 0.01031) {
-	  passID = true;
+
+    if(correctSizeVecs){
```

Comment:
should never get to this point with incorrect vector sizes (`correctSizeVecs` doesn't even need to be a member variable if above recommendation is followed)